src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -356,7 +357,9 @@ public function newAction()
         /** @var Product $product */
         $product = $productAdapter->getProductInstance();
         $product->id_category_default = $context->shop->id_category;
-        $product->id_tax_rules_group = 0;
+        /** @var TaxRuleDataProvider $taxRuleDataProvider */
+        $taxRuleDataProvider = $this->get('prestashop.adapter.data_provider.tax');
```

Comment:
oh yeah, I got it :) it's to be able to find the related service :)

Reply:
Yeah. Actually, I didn't really need it because PhpStorm is able to resolve the `$this->get()` calls. But I could not assume any contributor has such a good IDE :)